projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-results/search-ai-results.component.html

```diff
@@ -10,10 +10,55 @@
     class="aca-page-layout-content"
     *ngIf="!hasError">
     <div class="aca-search-ai-results-container">
-      <div
-        class="aca-search-ai-results-container-query"
-        data-automation-id="aca-search-ai-results-query">
-        {{ searchQuery }}
+      <div class="aca-search-ai-results-container-wrapper"
+           *ngIf="!editing"
+           #toggleIcon="toggleIcon"
+           data-automation-id="toggle-edit-trigger-container"
+           adf-toggle-icon>
+        <div
+          class="aca-search-ai-results-container-wrapper-query"
+          data-automation-id="aca-search-ai-results-wrapper-query">
+          {{ searchQuery }}
+          <ng-container
+            *ngTemplateOutlet="editButtonTemplate;context:{toggleIcon, disabled: false}">
+          </ng-container>
+        </div>
+      </div>
+      <div class="aca-search-ai-results-container-wrapper"
+           *ngIf="editing">
+        <mat-form-field subscriptSizing="dynamic"
+                        class="aca-search-ai-results-container-wrapper-form-field"
+                        appearance="outline">
+          <textarea
+            matInput
+            class="aca-search-ai-results-container-wrapper-form-field-input"
+            data-automation-id="toggle-edit-input-textarea"
+            [attr.aria-label]="'SEARCH.INPUT.ARIA-LABEL' | translate"
+            [(ngModel)]="searchQuery"
+            (keyup.enter)="updateInput()"
+            [placeholder]="'SEARCH.INPUT.PLACEHOLDER' | translate"
+            autocomplete="off"
+          ></textarea>
+          <ng-container
+            *ngTemplateOutlet="editButtonTemplate;context:{toggleIcon: {}, disabled: true}">
+          </ng-container>
+          <button
+            (click)="cancelUpdateInput()"
+            mat-flat-button
+            [attr.aria-label]="'CORE.METADATA.ACCESSIBILITY.EDIT' | translate"
```

Comment:
We should not use translations from metadata here as it's not metadata. For metadata we have different place in application. Please add separate translation key or reuse different if there is some